manifests/deployment.yaml

```diff
@@ -0,0 +1,24 @@
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: objectstorage-controller
+  namespace: objectstorage-system
+spec:
+  replicas: 1
+  strategy:
+    rollingUpdate:
```

Comment:
Unless leader election is implemented, this could cause trouble :smiley:

Reply:
We have leader election